android/src/main/java/com/swmansion/rnscreens/ext/ViewExt.kt

```diff
@@ -0,0 +1,18 @@
+package com.swmansion.rnscreens.ext
+
+import android.view.View
+import android.view.ViewGroup
+
+fun View.recycle(): View {
```

Comment:
What is it and why is it in this PR?

Reply:
Cherrypicked one commit too much. 

https://github.com/software-mansion/react-native-screens/pull/1887/commits/ee40686d50d566f80fca4f4947b6278b50710c08